_data/simple-icons.json

```diff
@@ -1416,7 +1416,11 @@
         {
             "title": "Canonical",
             "hex": "77216F",
-            "source": "https://design.ubuntu.com/downloads/"
+            "source": "https://assets.ubuntu.com/v1/d8a99c78-canonical-symbol-set-web.zip",
```

Comment:
We prefer link to pages that includes file instead of link to direct files. In this case, it would be https://design.ubuntu.com/downloads/

This way, if the file name or path changes, the link is still working.

Reply:
I made requested change in https://github.com/simple-icons/simple-icons/pull/6323/commits/c1fe478d3cf92eef4de130ebf699ba281b0804c9